server/src/util/Storage.ts

```diff
@@ -0,0 +1,136 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Hvy Industries. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *  "HVY", "HVY Industries" and "Hvy Industries" are trading names of JCKD (UK) Ltd
+ *--------------------------------------------------------------------------------------------*/
+
```

Comment:
Can you refactor this into a class called `Storage` please?

Making this a class would better fit the style of the rest of the server code.